.ci/format.sh

```diff
@@ -30,6 +30,10 @@ format() {
     
     # NOTE: once the files of a directory are (almost) fully covered, consider moving it to allowlist_dir instead
     local allowlist_file="
+        src/Equations/elastic/Model/datastructures.hpp
+        src/Equations/viscoelastic2/Model/datastructures.hpp
```

Comment:
What about viscoelastic(1)?

Reply:
Do we still support `viscoelastic` (1)? Otherwise, yes; I can change it.

Reply:
Wait—we don't have a `Model/datastructures.hpp` for viscoelastic (1). :)

Reply:
Yes, viscoelastic(1) is a prototype for a non-stiff pde with source term.

Reply:
We don't have a datastructure.hpp for viscoelastic(1), we just use the same as for viscoelastic2.